fail2ban/tests/filtertestcase.py

```diff
@@ -251,21 +257,21 @@ def testIgnoreIPNOK(self):
 
 	def testIgnoreIPCIDR(self):
 		self.filter.addIgnoreIP('192.168.1.0/25')
-		self.assertTrue(self.filter.inIgnoreIPList('192.168.1.0'))
-		self.assertTrue(self.filter.inIgnoreIPList('192.168.1.1'))
-		self.assertTrue(self.filter.inIgnoreIPList('192.168.1.127'))
-		self.assertFalse(self.filter.inIgnoreIPList('192.168.1.128'))
-		self.assertFalse(self.filter.inIgnoreIPList('192.168.1.255'))
-		self.assertFalse(self.filter.inIgnoreIPList('192.168.0.255'))
+		self.assertTrue(self.filter.inIgnoreIPList(IPAddr('192.168.1.0')))
```

Comment:
so the same automagic conversion could have been done in `inIgnoreIPList` (which would be at least the 4th place of the `x = x if isinstance(x, ...) else X(x)` hopefully warranting a helper function ;) e.g. `asip` or alike ;)).  That would avoid all the IPAddr'ing here, making diff smaller, less duplication, etc